### PR TITLE
Add Turbo Timber Evolution airframe build

### PR DESCRIPTION
**Pull request by chris1seto**
Add my build of a Turbo Timber Evolution with a Holybro Pixhawk 4 Mini to the airframes list

### CODE REVIEW

**Comment by dagar** [comment]
Pictures would be great if you have them.

**Comment by chris1seto** [comment]
> Pictures would be great if you have them.

Yup, I have a ton of photos I'm planning to add

**Comment by hamishwillee** [comment]
@chris1seto Thanks very much - we can certainly use a lot more information on frames, and in particular the FW ones.

I won't review this until you put it out of draft, but a couple of notes about what I look for (not all will apply, just stuff to think about).

1. A good structure that people can follow logically to build and configure the frame - so typically "overview", Bill of materials, if using a kit "whats in the kit", "assembly", PX4 configuration, Cool videos.
   - A good MC examples is http://docs.px4.io/main/en/frames_multicopter/holybro_x500V2_pixhawk5x.html
   - I haven't provided a good Fixed wing example, because there isn't one.
2. One flaw in all our build logs is that they explain "what I did" but not usually "why I did it"? As a result, I can copy what you did, but I don't necessarily learn how to extend them.
   - So if there are particular design decisions you made would be great to catch them. THings like "I chose this motor because of Xyz" or "Here I twisted the wires because of XyZ", or "I mounted the GPS here as it was the only location available - but I would have liked to have moved it to ABC".
3. All the old logs use the old airframe selection and then perhaps point to tuning guides. I see you are thinking of providing a parameter file (that presumably can go on the SD card?). Would be useful IFF it makes sense to point to actuators setup page/show the setup and perhaps explain it. Also to explain what tuning guides you followed.

This will also need to be added to the /en/SUMMARY.md.

I'm here to help when you get a bit further along.

**Comment by chris1seto** [comment]
Thanks @hamishwillee ! 

I still have a bit more writing to go, just been swamped at work. I agree with all of your points, I have been trying to hit those items already actually. 

And yes, I want to write up a quick howto of how to bring up a new airframe such as this one, though that might be more of a general guide.

**Comment by hamishwillee** [comment]
Great. Looking forward to seeing what you come up with!

**Comment by chris1seto** [comment]
@hamishwillee What is the best tense to write the guide in? How *I* built it (past tense) or how it should be built (future tense)?

**Comment by hamishwillee** [comment]
@chris1seto I prefer future tense: "Start by doing X". If you feel you had to do it in past tense that is fine (and sometimes people do). What is not OK is to mix them :-).

**Comment by hamishwillee** [comment]
The image fiekld.png looks pretty cool. Any reason you chose not to include it in the docs? I it isn't used please remove it because otherwise it adds to the cost of installing docs locally from github.

**Comment by hamishwillee** [comment]
@chris1seto I have started to subedit this a bit and uploaded my changes. Many of them are "hamish conventions" that you don't need to particularly learn from - for example, I put all sentences on their own lines. Why? Well because it _sometimes_ makes it easier for translators but it always makes it easy to add very specific review comments.

One of the things I have done is started to go through the images and reduce them in size. There are two reasons for this:
1. A very large file size means larger time to sync the docs if you're working locally, and makes the page more heavyweight for someone who wants to download it. Absolutely fine with a huge image if that is what is required, but it almost never is. Normally I consider an image worth looking at if it is more than 200Kb. 
2. An image should show what it needs to show. 
   - So for example in a picture of a pitot there is no need for everything around it unless you want to show scale etc. Cut out all the crap so that people can see the bits of interest - i.e. the pitot connections, the cable ties, the heat glue etc.
   - By contrast you need to keep some of the "framing" in flight controller installation because you need that to provide context of where things go.
   
In addition:
- Landscape works better on landscape computer screens - so generally if an image can be landscape, I flip it. All the image is usually visible to much the same extent, but now you can get to the content quicker.

I'll continue that, so most of this is FYI

File: en/frames_plane/turbo_timber_evolution.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
+
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:61]: I added the words here to make the image "accessible" - that means if someone has impaired vision they get a description of what the useful part of the image is showing.

Now I'm not sure how many such people will be using these docs but it is still useful because it helps you understand why you are showing the image.

File: en/frames_plane/turbo_timber_evolution.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)
+
+The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
+Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
+A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch were then be used to mount the FPV pod.
+
+## Pitot Pod
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:69]: This is the only section I have actually subedited so far. Some comments one what/why below.

At high level though, this was a set of images and some description that were pretty hard to follow. Explaining what you are doing and why, and the alternatives makes things much easier.

File: en/frames_plane/turbo_timber_evolution.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+A airspeed sensor is highly recommended for use on fixed wing vehicles.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:71]: It is worth assuming that your audience are completely new to aircraft. So why do they need to do this? Well, because an airspeed sensor is needed by fixed wing aircraft.

- If it is mandatory? If so we should update to say that.
- We could/should link to the airspeed sensor page for more information and options: http://docs.px4.io/main/en/sensor/airspeed.html

File: en/frames_plane/turbo_timber_evolution.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+This build uses a MS4525DO differential pressure module and pitot tube housed in a 3D printed pod that has a hardpoint hanger for connecting it to the wing.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:72]: You have done something special/different here. Most people just buy and off the shelf job while you have created your housing by 3d printing - presumably to get a more professional mounting to the wing.
- What I have done is made it clear that you have done this because it isn't obvious from the parts list unless you know what a pitot is and the sensor. 
- We should perhaps say "why" - ... because this results in a better X, Y, Z.
- We should link to alternative sensor options "You can use any of the sensors here if you prefer".

File: en/frames_plane/turbo_timber_evolution.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+![Pitot pod/tube sitting on a desk](../../assets/airframes/fw/turbo_timber_evolution/pitotpod1.jpg)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:74]: I guess this image is just to show the final structure. If you had a more landscape image that might be better - i.e. it takes up a lot of unnecessary height.

File: en/frames_plane/turbo_timber_evolution.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+
+
+Inside the pitot pod, the MS4525DO differential pressure sensor is housed and is connected to the pitot/static tube with a short length of tubing.
+Zipties are used as hoseclamps to prevent the tubing from backing off.
+The i2c and power leads were soldered directly to the MS4525 module, and then hotglue was used to mechanically reinforce the connections.
+
+![Pitot pod opened up showing wires pitot tube connectors and wires soldered and hot-glued in place](../../assets/airframes/fw/turbo_timber_evolution/pitotpod2.jpg)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:81]: Note, here I have split the description and the image - I think that makes it much easier to work out what the image is showing you, and why you provided it.

I am not proposing you do this, but what is even better than this is instructional steps like this:

1. Insert the wiring etc
    _image here_
2. Connect on wing to right of XYZ
    _image here_
3. And so on.

The reason is that if you have steps people can follow, they can't miss anything and neither can you. It makes the whole thing very structured and turnkey.

However as long as the information is here and the steps that need to be taken can be inferred, this is still able to pass the test "can people duplicate the work"

File: en/frames_plane/turbo_timber_evolution.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+The pitot/static differential pressure sensor was mounted on the wing (outside the radius of the prop) using a 3D printed "hanger" glued to the leading edge of the wing.
+An M2 screw and nylock hold the pod to the hanger.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pitotpodinstalled.png)
+
+These 4 leads were then taped to the underside of the wing running back to the Pixhawk 4 Mini.
+
+The lid of the pitot pod was initially taped in place to allow the setup to be tested and modified if needed after first flight.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:90]: FYI, just makes it a bit easier to read.

File: en/frames_plane/turbo_timber_evolution.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+[A Quick Flight](https://youtu.be/1DUV7QjcXrA)
+
+### Flight Logs
+
+[Evening Flight (Includes video)](https://review.px4.io/plot_app?log=d3f2c1f9-f802-48c1-ab5d-3983fc8b8719)
+
+### Parameter File
+
+[Parameters](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/fw/turbo_timber_evolution/tteparams.params)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:203]: FYI, not that you can't download files using the link format to assets - in the built version that ends up taking you to an empty page. The workaround is to link to the raw file - this will download from main when the docs are merged.

File: en/frames_plane/turbo_timber_evolution.md [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+[Spot Landing](https://www.youtube.com/watch?v=vMFCi3G5s6E)
+
+[A Quick Flight](https://youtu.be/1DUV7QjcXrA)
+
+### Flight Logs
+
+[Evening Flight (Includes video)](https://review.px4.io/plot_app?log=d3f2c1f9-f802-48c1-ab5d-3983fc8b8719)
+
+### Parameter File
+
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:202]: Please add instructions or link to instructions on how these would be applied by a user who is just getting started?

Note, this is a lot of params - is it all the params or just the Diff vs the generic frame?

File: en/frames_plane/turbo_timber_evolution.md [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+### Performance
+
+- Stall speed (no flaps): 14MPH indicated
+- Cruise speed: 35-65MPH
+- Takeoff roll (with full flaps): < 10ft
+- Endurance: ~24 minutes on 5.2Ah 4s2p LiIon, ~12 minutes on 3.6Ah 4S LiPo
+
+### Videos
+
+[Spot Landing](https://www.youtube.com/watch?v=vMFCi3G5s6E)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:193]: do you want these to be links or displayed as video in place? We can support either?

File: en/frames_plane/turbo_timber_evolution.md [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+
+These 4 leads were then taped to the underside of the wing running back to the Pixhawk 4 Mini.
+
+The lid of the pitot pod was initially taped in place to allow the setup to be tested and modified if needed after first flight.
+After the first flight, the lid was hot glued in place.
+
+## Flight Computer Installation
+
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pre_mount_install.jpg)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:96]: I made this image landscape. If you had the plane in front of you, would it be easy to tell which way is the front? If not, should we (you) add an arrow to the image showing the orientation?

File: en/frames_plane/turbo_timber_evolution.md (hunk 3 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:71]: Got it, I'll add to this section to yield more information. I don't know if you are familiar with FW, but no, it's not mandatory, I just did it for better state information since this was supposed to be a development/test platform.

File: en/frames_plane/turbo_timber_evolution.md (hunk 8 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:203]: Got it! Thanks!

File: en/frames_plane/turbo_timber_evolution.md (hunk 9 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:202]: I'll add a blurb on how to load them. As far as the scope of the params, this is a whole dump of the entire, I could do the airframe dump. I guess that'd probably be better to remove the calibration params.

File: en/frames_plane/turbo_timber_evolution.md (hunk 10 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:193]: Video in place would be great! I googled around and didn't see any immediate way to do this. What's the incantation?

**Comment by chris1seto** [comment]
Thank you very much for the review @hamishwillee !! I will submit a set of edits sometime today according to your comments.

File: en/frames_plane/turbo_timber_evolution.md (hunk 11 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:97]: It should be very easy to tell front vs back because of the servos. I don't think this image is ambiguous if you had the plane sitting in front of you.

File: en/frames_plane/turbo_timber_evolution.md (hunk 10 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:196]: Nvm, I found an example and added this. Thanks for the tip!

File: en/frames_plane/turbo_timber_evolution.md (hunk 9 above)
Comment (hamishwillee) [reply en/frames_plane/turbo_timber_evolution.md:209]: Yes, we should point to calibration docs instead in the basic config page and state that people should still do that for their setup before flying.

**Comment by ryanjAA** [comment]
This is awesome! A new FW build guide was definitely missing!

**Comment by chris1seto** [comment]
@hamishwillee At this point, I think I am done with adding new content. Are there any more review items you'd like to address which hare blocking a merge? Thanks!!

File: en/frames_plane/turbo_timber_evolution.md [hunk 12]
```diff
@@ -0,0 +1,213 @@
+# Turbo Timber Evolution (TTE) Build
+
+![Turbo Timber Evolution Closup in a field](../../assets/airframes/fw/turbo_timber_evolution/field_overview1.jpg)
+![Turbo Timber Evolution Closup](../../assets/airframes/fw/turbo_timber_evolution/field.png)
+
+The Turbo Timber Evolution is a model sold by Horizon Hobby originally intended for classic line-of-sight RC model flying.
+This model is designed to excel at STOL flying and has a number of notable traits that also make it an ideal candidate to be converted into an FPV PX4 platform.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:7]:
```suggestion
This model is designed to excel at [STOL](https://en.wikipedia.org/wiki/STOL) flying and has a number of notable traits that also make it an ideal candidate to be converted into an FPV PX4 platform.

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+* Long flight times (with Liion battery option > 24 minutes)
+
+# Build
+
+## Parts List
+
+- [Turbo Timber Evolution PNP (includes motor, servos, esc, etc, all fully installed)](https://www.horizonhobby.com/product/turbo-timber-evolution-1.5m-pnp-includes-floats/EFL105275.html#)
+- [80A Plush-32 ESC](https://hobbyking.com/en_us/turnigy-plush-32-80a-2-6s-brushless-speed-controller-w-bec-rev1-1-0.html)
+- [Pixhawk 4 Mini/GPS/Power module/ set](https://shop.holybro.com/pixhawk4-mini_p1120.html)
+- SIK telemetry radio
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:42]:
```suggestion
- [SIK telemetry radio](../telemetry/sik_radio.md)

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+* Air data provided by wing slung pitot static pod
+* Long flight times (with Liion battery option > 24 minutes)
+
+# Build
+
+## Parts List
+
+- [Turbo Timber Evolution PNP (includes motor, servos, esc, etc, all fully installed)](https://www.horizonhobby.com/product/turbo-timber-evolution-1.5m-pnp-includes-floats/EFL105275.html#)
+- [80A Plush-32 ESC](https://hobbyking.com/en_us/turnigy-plush-32-80a-2-6s-brushless-speed-controller-w-bec-rev1-1-0.html)
+- [Pixhawk 4 Mini/GPS/Power module/ set](https://shop.holybro.com/pixhawk4-mini_p1120.html)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:41]:
```suggestion
- [Pixhawk 4 Mini](https://shop.holybro.com/pixhawk4-mini_p1120.html) (with GPS and Power module)

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+  - Pitot static pod and wing hardpoint hanger
+- Misc hardware: M3 hardware (standoffs, washers, bolts), XT30 connector, hot glue, heatshrink, Molex Microfit connectors
+- Silicone wiring (14awg for high current, 16awg for low current, 22awg for low power and signals)
+- 3.6Ah 4S LiPo OR 4s2p 18650 LiIon
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:59]:
```suggestion
The vehicle comes out of the box near complete.
Servos and linkages have already been installed, and the only real remaining work is installing the landing gear and horizontal stabilizer.
For this portion of the assembly, simply follow the manual.

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 16]
```diff
@@ -0,0 +1,213 @@
...
+
+## A Note On Power Systems
+[Some reports](https://www.rcgroups.com/forums/showthread.php?3904021-NEW-E-flite-Turbo-Timber-Evolution-1-5m-%C2%96-Smartest-Most-Capable-Durable-Timber-Yet/page50) have indicated that the stock ESC bundled with the airplane has an issue with overheating.
+Because I knew this build would be especially heavy, and therefore would likely demand higher average power from the ESC, I opted to replace the stock 60A ESC with an 80A Turnigy PLUSH-32.
+At the time of writing, I was also in the process of replacing the stock motor with a [higher rated power motor](https://hobbyking.com/en_us/turnigy-aerodrive-sk3-3548-840kv-brushless-outrunner-motor.html), but had not yet completed a test flight.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:64]:
```suggestion
## A Note On Power Systems

[Some reports](https://www.rcgroups.com/forums/showthread.php?3904021-NEW-E-flite-Turbo-Timber-Evolution-1-5m-%C2%96-Smartest-Most-Capable-Durable-Timber-Yet/page50) have indicated that the stock ESC bundled with the airplane has an issue with overheating.

As this build will be especially heavy, and therefore likely demand higher average power from the ESC, the default 60A ESC has been replaced with an 80A Turnigy PLUSH-32.
The stock motor will also be replaced with a [higher power motor](https://hobbyking.com/en_us/turnigy-aerodrive-sk3-3548-840kv-brushless-outrunner-motor.html) (this has not yet been flight-tested).

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+That design goal meant that naturally balanced controls representative of a "vanilla" airplane were desirable.
+Because classic RC planes are usually designed to be hand flown with no computer augmented flight controls, they tend to be specifically designed to be well trimmed and balanced out of the box.
+These airplanes also have more attention devoted to making sure they handle well in the air.
+While it's possible to fly even the most simple foamboard airplane, a lot of nuance in airborne handling can be finessed with a bit more engineering effort.
+This airplane is a premium example of that, with features like frise ailerons to minimize adverse yaw.
+
+Key Build Features
+* Easy overall build with minimal airframe setup
+* Easy access to Pixhawk USB and debug connector
+* FPV with camera pan mount
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:31]:
```suggestion
* [First Person View (FPV)](https://en.wikipedia.org/wiki/First-person_view_(radio_control)) with camera pan mount

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 18]
```diff
@@ -0,0 +1,213 @@
...
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)
+
+![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)
+
+The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
+Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
+A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch, were then be used to mount the FPV pod.
+
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:75]: I like text first.

```suggestion
## FPV Pod

The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch, were then be used to mount the FPV pod.

![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)

![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)

:::note
PX4 itself does not integrate with the FPV system, so this has no connection with the flight controller.
However it does need to be powered.
:::

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 19]
```diff
@@ -0,0 +1,213 @@
+# Turbo Timber Evolution (TTE) Build
+
+![Turbo Timber Evolution Closup in a field](../../assets/airframes/fw/turbo_timber_evolution/field_overview1.jpg)
+![Turbo Timber Evolution Closup](../../assets/airframes/fw/turbo_timber_evolution/field.png)
+
+The Turbo Timber Evolution is a model sold by Horizon Hobby originally intended for classic line-of-sight RC model flying.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:6]:
```suggestion
# Turbo Timber Evolution (TTE) Build

The Turbo Timber Evolution is a model sold by Horizon Hobby originally intended for classic line-of-sight RC model flying.

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 20]
```diff
@@ -0,0 +1,213 @@
+# Turbo Timber Evolution (TTE) Build
+
+![Turbo Timber Evolution Closup in a field](../../assets/airframes/fw/turbo_timber_evolution/field_overview1.jpg)
+![Turbo Timber Evolution Closup](../../assets/airframes/fw/turbo_timber_evolution/field.png)
+
+The Turbo Timber Evolution is a model sold by Horizon Hobby originally intended for classic line-of-sight RC model flying.
+This model is designed to excel at STOL flying and has a number of notable traits that also make it an ideal candidate to be converted into an FPV PX4 platform.
+
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:8]: Add just one image back - well use the other below.
```suggestion

![Turbo Timber Evolution Closup](../../assets/airframes/fw/turbo_timber_evolution/field.png)

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 21]
```diff
@@ -0,0 +1,213 @@
...
+## Build Results & Performance
+
+Overall, this build was a success.
+Even with the added weight of the Pixhawk 4 Mini installation, the airplane balances well and has plenty of power to retain its original STOL characterisitics.
+PX4 is easily capable of stabilizing the airplane and fine tuning of the rate loops were accomplished using the fixed wing autotune module.
+The results of tuning can be found in the parameter file attached below.
+In testing I found takeoffs can be as short as only 10ft using no flaps.
+I use full flaps on landing to slow the otherwise slippery airframe.
+
+![Turbo Timber Evolution Field Setup](../../assets/airframes/fw/turbo_timber_evolution/field_setup.jpg)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:188]:
```suggestion
![Turbo Timber Evolution Closup in a field](../../assets/airframes/fw/turbo_timber_evolution/field_overview1.jpg)
![Turbo Timber Evolution Field Setup](../../assets/airframes/fw/turbo_timber_evolution/field_setup.jpg)

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 22]
```diff
@@ -0,0 +1,229 @@
...
+Servo endpoints were obtained by using a servo tester to determine the servo PWM pulse width to reach the max travel of each surface in each direction.
+
+
+### Config & Debug
+
+Access to the Pixhawk 4 Mini requires removal of the upper mount.
+While this isn't too difficult, it was a consideration for wanting to streamline debugging in the field.
+A short right angle USB micro extension was used to allow easy access to the Pixhawk 4 Mini's USB interface.
+The USB-A end of this cable was left dangling in the battery bay.
+Similarly, a JST PH to std spaced headers adapter was made, and it was also left easily accessible in the battery bay.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:168]: Would be good to have a picture of the adapters. Wouldn't block this going in though.

File: en/frames_plane/turbo_timber_evolution.md [hunk 23]
```diff
@@ -0,0 +1,229 @@
...
+While the 3.6Ah LiPo is inexpensive, nearly twice the flight time (24 minutes vs 12 minutes) can be acheived with the Upgrade Energy Liion pack.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/batteries.jpg)
+
+### Servos
+
+Servos were wired to the flight computer in order of aileron, elevator, rudder, throttle, flaps, and FPV pan.
+An additional power plug for the lighting system needs to also be installed, but it does not carry a servo signal so it can be put on any spare channel.
+
+The [Acutator Configuration](../config/actuators.md) screen is shown below.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:155]: FYI added this link to help people understand the image.

File: en/frames_plane/turbo_timber_evolution.md [hunk 24]
```diff
@@ -0,0 +1,229 @@
...
+
+Another custom cable was made to connect the Caddx Vista to the FCU UART and power.
+A Molex microfit was added close to the Vista so that it could be easily disconnected without needing to gain access to the Pixhawk.
+The `msp_osd` module is used to stream telemetry to the Caddx Vista which can be seen on the DJI Goggles with the "custom OSD" feature enabled.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod.jpg)
+
+#### SIK Telemetry Radio
+
+The plastic case of the SIK telemetry radio was removed to reduce weight and decrease volume of the module.
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:189]: Image would be nice showing this - but not essential

File: en/frames_plane/turbo_timber_evolution.md [hunk 25]
```diff
@@ -0,0 +1,232 @@
...
+
+Access to the Pixhawk 4 Mini requires removal of the upper mount.
+While this isn't too difficult, it was a consideration for wanting to streamline debugging in the field.
+A short right angle USB micro extension was used to allow easy access to the Pixhawk 4 Mini's USB interface.
+The USB-A end of this cable was left dangling in the battery bay.
+Similarly, a JST PH to std spaced headers adapter was made, and it was also left easily accessible in the battery bay. 
+
+### Peripherals
+
+#### ExpressLRS RX
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:172]:
```suggestion
#### RC Receiver

```

File: en/frames_plane/turbo_timber_evolution.md [hunk 26]
```diff
@@ -0,0 +1,232 @@
...
+
+A custom cable was made to connect the ExpressLRS RX ([RC Reciever](../getting_started/rc_transmitter_receiver.md)) to the Pixhawk 4 Mini.
+
+Because the Pixhawk 4 Mini has limited uarts, the RX was connected to RC input which does not have a TX pin.
+This means that the RX will only send control data to the FCU but telemtry cannot be sent to the RX from the FCU.
+Heatshrink was used to secure the dupont connector of the cable such that it cannot back out off the headers of the ExpressLRS RX.
+
+#### Caddx Vista (FPV camera)
```
Comment (hamishwillee) [inline en/frames_plane/turbo_timber_evolution.md:180]:
```suggestion
#### FPV Camera (Caddx Vista)

```

**Comment by hamishwillee** [comment]
@chris1seto I've subedited a bit more to my taste 
- mostly things like cross linking to other resources
- - prefer to have text explaining images, rather than showing an image and then explaining it. 
- I also prefer to get to content quick, so even though I love the image of the vehicle on a car, I moved it further down in the doc.
- I like to name things - so you refer to parts like "ExpressLRS RX" - but if you're not familiar with the parts it isn't clear what the section is about - so I renamed those.

The whole thing holds together pretty well I think and can be followed by someone with the bits in front of them (maybe some problems working out a few of the custom cables, but that's life). The videos are very cool - excellent showing off of the main things people are interested in - such as landing.

In particular, really appreciate you adding "reasons" for why you did things - e.g. removing the case of the Telemetry radio.

Merging!

Note, I made one or more

**Comment by hamishwillee** [comment]
PS If I screwed up stuff in my edit, PR welcome.

**Comment by hamishwillee** [comment]
Curses, the params file is empty!! @chris1seto can you upload a new version?

File: en/frames_plane/turbo_timber_evolution.md (hunk 22 above)
Comment (chris1seto) [reply en/frames_plane/turbo_timber_evolution.md:168]: Yeah I wished I had taken one one. Unfortunately, they're buried in the airplane now, so that was the only reason I didn't include a shot.

**Comment by chris1seto** [comment]
Thanks a ton for the hge help Hamish, yes, I'll fix the params file and there are a few other tiny edits. Note for the FPV system, PX4 does actually support integration with this! Myself and another guy wrote support for that in the msp_osd driver, which was merged a few months ago. Also, the new power system has since been test flown.

File: en/frames_plane/turbo_timber_evolution.md (hunk 22 above)
Comment (hamishwillee) [reply en/frames_plane/turbo_timber_evolution.md:168]: Bugger!

**Comment by hamishwillee** [comment]
Thank you @chris1seto - really appreciate you taking the time to create this. The only one one is so old and rubbish I have been wanting to remove it forever - now I can :-)

W.r.t. FPV integration, I'll have a look at what you write in the other PR. My concern here is discoverability - a new feature/function that no one else can use because no one knows about it. 
I think we need an FPV page that says what is possible/what you can do in terms of integration, and links to supported systems. Does that make sense to you? Can you help with it> This would live below peripherals - see http://docs.px4.io/main/en/peripherals/
